config/webpack.common.js

```diff
@@ -2,17 +2,17 @@
  * @author: @AngularClass
  */
 
-var webpack = require('webpack');
-var helpers = require('./helpers');
+const webpack = require('webpack');
+const helpers = require('./helpers');
```

Comment:
I would still use `var` since the user will have an error if they're not node 4+ I'll be fine with it if we have another way to warn the user to update their node version before this error